src/www/app/fake_tunnel.ts

```diff
@@ -24,12 +24,12 @@ export class FakeOutlineTunnel implements Tunnel {
 
   constructor(public readonly id: string) {}
 
-  private playBroken(hostname?: string) {
```

Comment:
Thanks @andreevgy for the contribution. I would suggest to use tag instead of host IP because we might introduce multiple mocked broken/unreachable tunnels.

Reply:
Sorry for long reply, with busy with work. But tag does not get passed on this level, so it's not accessible and because of it it's broken. Do we want to pass it here?

Reply:
Got it, then I think it's OK to use the IP address here. But I would recommend to `export` them as `const`s instead of using them as magic strings, and use these `const`s both here and in [main.ts](https://github.com/Jigsaw-Code/outline-client/blob/9105e53152f36b6d0bf4beab534f4219ae93aeec/src/www/app/main.ts#L63). Maybe:

```
export const FAKE_BROKEN_HOST = "...";
export const FAKE_UNREACHABLE_HOST = "...";
```

Reply:
I don't understand. If you can specify the hostname, you can just specify a domain name with the corresponding patterns.

Reply:
For example: "broken.example.com" or "unreachable.example.com".
This is way better, as it lets use have multiple servers and trigger many behaviors, with the name reflecting the behavior.

Reply:
Can you clarify what you are trying to accomplish?
You can already test errors by adding servers with the appropriate host names.

Reply:
You are right that it can be tested by adding servers with appropriate host names. But there are default mock ones named "broken" and "unreachable" that do not behave like that. That's what I tried to fix first by checking IP address because tag is not passed all the way here. I thought you meant that instead of ip address we can set certain hostname to mock broken server and that's why I am asking if I should open another MR or do changes here. Because default mock servers that I see now first time launching dev environment are "working" just fine.